httpserver/http_connection.cpp

```diff
@@ -504,6 +483,34 @@ void connection_t::process_retrieve_all() {
     response_.result(http::status::ok);
 }
 
+bool connection_t::check_correct_swarm(std::string pub_key) {
```

Comment:
The name suggests that it is a simple check, while it has a lot of side effects. I would either leave `if (service_node_.is_pubkey_for_us(pub_key)) {...}` and add a `handle_incorrect_swarm` function, or rename this function to something like `check_and_handle_incorrect_swarm`.

Reply:
Oh right that makes a log more sense